[2024-12-19T05:44:49.525+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T05:44:49.531+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2024-08-08T00:00:00+00:00 [queued]>
[2024-12-19T05:44:49.534+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2024-08-08T00:00:00+00:00 [queued]>
[2024-12-19T05:44:49.534+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T05:44:49.541+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2024-08-08 00:00:00+00:00
[2024-12-19T05:44:49.544+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4783) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T05:44:49.544+0000] {standard_task_runner.py:72} INFO - Started process 4791 to run task
[2024-12-19T05:44:49.544+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2024-08-08T00:00:00+00:00', '--job-id', '1110', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpyvtdonl5']
[2024-12-19T05:44:49.545+0000] {standard_task_runner.py:105} INFO - Job 1110: Subtask first_task
[2024-12-19T05:44:49.566+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2024-08-08T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T05:44:49.595+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-08T00:00:00+00:00'
[2024-12-19T05:44:49.595+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T05:44:49.601+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T05:44:49.601+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T05:44:49.604+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T05:44:49.605+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T05:44:49.605+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T05:44:49.614+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T05:44:49.615+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2024-08-08T00:00:00+00:00, execution_date=20240808T000000, start_date=20241219T054449, end_date=20241219T054449
[2024-12-19T05:44:49.645+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T05:44:49.650+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T05:44:49.651+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
